src/containers/chat/index.js

```diff
@@ -4,10 +4,7 @@ import Existing_Chat from './existing_chat'
 import Account_Screen_New from './account_screen_new'
 import TextField from 'material-ui/TextField';
 import { withStyles } from 'material-ui/styles';
-import Paper from 'material-ui/Paper';

```

Comment:
Were these never used? We should have seen it then in the previous PRs, shame on us.